runtime/ptrace_check.cc

```diff
@@ -57,18 +70,16 @@ static bool DebuggedIOS(const flutter::Settings& vm_settings) {
   return proc->kp_proc.p_flag & P_TRACED;
 }
 
-void EnsureDebuggedIOS(const flutter::Settings& vm_settings) {
-  if (DebuggedIOS(vm_settings)) {
-    return;
-  }
-
-  if (ptrace(PT_TRACE_ME, 0, nullptr, 0) == -1) {
+static bool EnableTracingManually(const Settings& vm_settings) {
+  if (::ptrace(PT_TRACE_ME, 0, nullptr, 0) == -1) {
```

Comment:
Add some code comments for readers (e.g. these 2 ifs are expected to fail on iOS 14, the rest should let people jit on iOS 13 etc)

Reply:
I don't think we can expect that. It's a private API and all implementation detail that can change at anytime without warning. I'd rather just be resilient to errors.